spot-contracts/contracts/_interfaces/IFeePolicy.sol

```diff
@@ -5,14 +5,16 @@ import { SubscriptionParams } from "./CommonTypes.sol";
 
 interface IFeePolicy {
     /// @param dr The current system deviation ratio.
+    /// @param dr_ The deviation ratio of the system after the operation is complete.
     /// @return The percentage of the mint perp tokens to be charged as fees,
     ///         as a fixed-point number with {DECIMALS} decimal places.
-    function computePerpMintFeePerc(uint256 dr) external view returns (uint256);
+    function computePerpMintFeePerc(uint256 dr, uint256 dr_) external view returns (uint256);
```

Comment:
Can we name these parameters something like `drPre` and `drPost`?

Reply:
Sounds good 👍